torch/serialization.py

```diff
@@ -1365,7 +1370,9 @@ def _load(zip_file, map_location, pickle_module, pickle_file='data.pkl', overall
 
     def load_tensor(dtype, numel, key, location):
         name = f'data/{key}'
-        if overall_storage is not None:
+        if torch._guards.detect_fake_mode(None) is not None:
+            storage = torch.UntypedStorage(numel, device='meta')
```

Comment:
```suggestion
            nbytes = numel * torch._utils._element_size(dtype)
            storage = torch.UntypedStorage(numel, device='meta')
```
should it be `numel` or `nbytes` here?